packages/react-components/src/components/chart/legend/table/legendTableAdapter.tsx

```diff
@@ -60,15 +62,21 @@ export const ChartLegendTableAdapter = ({
   >([]);
   const [trendCursors, setTrendCursors] = useState<TrendCursor[]>([]);
   const previousDatastreamInformation = useRef(
-    mapDataStreamInformation(datastreams, series, graphic)
+    mapDataStreamInformation(
+      datastreams,
+      series,
+      graphic,
+      options.visibleContent
```

Comment:
can we pull this out above ^ instead

ex: 
```
ChartLegendTableAdapter = ({
  datastreams,
  series,
  graphic,
  visibleContent, <==== here
  ...options
}: ChartLegendTableAdapterOptions)
```

Reply:
updated